modules/ROOT/pages/afm-view-monitor-history.adoc

```diff
@@ -0,0 +1,34 @@
+= View the Details of Past Executions of a Test
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:page-aliases: afm-view-test-history.adoc
+
+:imagesdir: ../assets/images
+
+You can review the details of past executions of a monitor's test. The *Test History* page for the monitor lists executions of the test. The *Execution Detail* page that appears for a selected execution displays the pairs of HTTP REST methods and endpoints that were tested.
+
+For example, the entries on *Execution Details* page look like these:
+
+[%hardbreaks]
+`DELETE "http://httpbin.org/delete"`
+`GET "http://httpbin.org/get"`
+`POST "http://httpbin.org/post"`
+
+When you expand an entry, you can see the response and the request, as well as the result for each assertion.
+
+== Procedure
+
+. In the list of your monitors, select the monitor that includes the test.
+. On the page that is opened, click the three dots next to the name of the monitor and select *View Test History*.
+. On the *Test History* page, click the list entry for the test execution that you want to see the details of.
```

Comment:
ditto: assume that the previous step locates the user where they should be, so we need not repeat:
```suggestion
. Click the list entry for the test execution that you want to see the details of.
```